hicommonwealth/commonwealth
# Add groups creation repeat migration

## Link to Issue
Closes: #6056

## Description of Changes
- Creates a duplicate of the `20231107161828-token-threshold-to-groups.js` migration, with a pre-step that truncates Memberships and Groups

## Test Plan
- Find a community that already has autogenerated groups
- Add a new group
- Run migration
- Confirm that new group is gone and autogenerated exist

## Deployment Plan
Run migration

## Other Considerations
N/A
